src/components/Tabs/Tabs.scss

```diff
@@ -351,6 +353,7 @@ $focus-state-box-shadow-color: rgba(0, 0, 0, 0.8);
       }
 
       @media (-ms-high-contrast: active) {
+        outline: 3px solid ms-high-contrast-color('text');
```

Comment:
Where does the `3px` come from? Is this informed by somewhere else in Polaris? If so, is there a way to share that code? If not, you should at least wrap `3pm` in the `rem()` function.

Reply:
this was not informed by Polaris, I just did a quick search and I see on `ActionList.scss` that they use `1px`: 

https://github.com/Shopify/polaris-react/blob/f1e8dcbfc049cca0fa1e9fa2323dcdc3c10b5897/src/components/ActionList/ActionList.scss#L209